smbprotocol/file_info.py

```diff
@@ -877,3 +877,23 @@ def __init__(self):
             )),
         ])
         super(FileFsVolumeInformation, self).__init__()
+
+
+class FileFsFullSizeInformation(Structure):
```

Comment:
I've tried to keep these in the order they are documented in the relevant MS docs, this should go before `FileFsObjectIdInformation`.